### ISSUE TITLE
Android 11 cannot be remotely controlled properly.

### ISSUE THREAD

**Issue by homuraLan** [rustdesk/rustdesk#7101]
### Bug Description

After enabling all permissions and accessibility access, still unable to control using the mouse.
Can watch Android screens.

### How to Reproduce

Install, then enable all permissions, and then start the service

### Expected Behavior

The mouse can be controlled normally

### Operating system(s) on local side and remote side

Windows 11 -> android 11

### RustDesk Version(s) on local side and remote side

v1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/96775034/bf257bd4-aa65-4b25-8db5-e70fe59cb8ae)

**Comment by rustdesk** [comment]
tracked by https://github.com/rustdesk/rustdesk/issues/3893
